README.md

```diff
@@ -77,3 +78,65 @@ In order to expose those ports outside the local environment, you can edit lines
       - "${SCNODE_WS_SERVER_PORT}:${SCNODE_WS_SERVER_PORT}"
       - "${SCNODE_REST_PORT}:${SCNODE_REST_PORT}"
    ```
+
+## Running a Forger Node 
+Should you want to run a forger node, please select "forger" when you run the init.sh script for the first time. Keep in mind it will take time for the Mainchain node and the sidechain node to fully sync the whole chains.
```

Comment:
How about this:
`If you wish to run a forger node, please choose 'forger' when you run the init.sh script for the first time. Please note that it will take time for both the Mainchain node and the Sidechain node to fully synchronize the entire chains.`